.github/workflows/sync-build-cache.yml

```diff
@@ -0,0 +1,102 @@
+name: Sync build cache
+
+permissions:
+    actions: write
+    contents: read
+
+on:
+    schedule:
+        - cron: '0 22 * * *'
```

Comment:
This runs everyday on `master` branch around the following time:
- 5:00 AM Malaysia time
- 1:00 AM Dubai time
- 12:00 AM Belarus time